image_client/client_tools.py

```diff
@@ -68,13 +68,15 @@ def main(args):
             BotanyImporter(paths=paths, config=botany_importer_config)
         elif args.collection == 'Botany_PIC':
             #get paths here
+            date_override = args.date
+            if date_override is not None:
+                rewrite_config_date(date_override, filepath="image_client/picturae_config.py")
```

Comment:
picturae_config.py is a python file; you don't need to parse it; Just import it and then use the variables therein.